packages/cli/src/commonlib/azureLogin.ts

```diff
@@ -465,24 +459,57 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
     }
   }
 
-  // TODO add login and select subscription logic later
-  getSelectedSubscription(triggerUI=false): Promise<SubscriptionInfo | undefined> {
-    if (AzureAccountManager.codeFlowInstance.account) {
+  async getSelectedSubscription(triggerUI = false): Promise<SubscriptionInfo | undefined> {
+    if (triggerUI) {
+      if (!AzureAccountManager.codeFlowInstance.account) {
+        await this.login(false);
```

Comment:
Why different from vscode, `true` or `false`?

Reply:
Now CLI will not ask user to confirm login or show learn more button.